specs/commandline/sign.md

```diff
@@ -31,9 +31,11 @@ Flags:
   -d,  --debug                      debug mode
   -e,  --expiry duration            optional expiry that provides a "best by use" time for the artifact. The duration is specified in minutes(m) and/or hours(h). For example: 12h, 30m, 3h20m
   -h,  --help                       help for sign
+       --id string                  key id (required if --plugin is set)
   -k,  --key string                 signing key name, for a key previously added to notation's key list.
   -p,  --password string            password for registry operations (default to $NOTATION_PASSWORD if not specified)
        --plain-http                 registry access via plain HTTP
+       --plugin string              signing plugin name
        --plugin-config stringArray  {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
```

Comment:
```suggestion
       --plugin-config stringArray  {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values. If --plugin is set, plugin configs for both key and sign should be included.
```
As I mentioned in previous comments, we might need to add this extra reminder for user. (Related spec: https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#plugin-configuration)

Reply:
IMO we don't need to mention this because it should be evident from not using the key name `-k`. If the key name is not passed then there is no plugin config for the key.